docs/001_develop/03_client-capabilities/008_interaction/radio-group.mdx

```diff
@@ -0,0 +1,169 @@
+---
+title: 'Radio Group'
+sidebar_label: 'Radio Group'
+id: client-interaction-radio-group
+keywords: [interaction, interactive, input, radio, radio-group, form, data]
+tags:
+ - interaction
+ - interactive
+ - input
+ - radio
+ - radio-group
+ - form
+ - data
+sidebar_position: 8
+---
+
+import RadioGroupDemo from '../../../../examples/ui/client-capabilities/interaction/radiogroup.js';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Radio group
+
+Use cases:
+* A radio group is a set of checkable buttons known as radio buttons.
+* No more than one button can be checked at a time.
+* When a user checks an unchecked button, the previously checked button is automatically unchecked.
+
+## Example
+
+<RadioGroupDemo />
+
+<Tabs defaultValue="genesis" values={[{ label: 'Genesis', value: 'genesis', }, { label: 'React', value: 'react', }, { label: 'Angular', value: 'angular', }]}>
+
+<TabItem value="genesis">
+
+Declaration
+```html
+<rapid-radio-group>
+  <rapid-radio value="one">One</rapid-radio>
+  <rapid-radio value="two">Two</rapid-radio>
+  <rapid-radio value="three">Three</rapid-radio>
+</rapid-radio-group>
+```
+
+Usage
+```typescript
+@customElement({
+  name: 'my-element',
+  template: html`
+    <rapid-radio-group
+      @change="${(x) => x.selectionChanged()}"
+      :value="${sync((x) => x.radioGroupValue)}"
+    >
+      <rapid-radio value="one">One</rapid-radio>
+      <rapid-radio value="two">Two</rapid-radio>
+      <rapid-radio value="three">Three</rapid-radio>
+    </rapid-radio-group>
+  `,
+})
+export class MyElement extends GenesisElement {
+  @observable radioGroupValue: string;
+  selectionChanged() {
+    console.log(this.radioGroupValue);
+  }
+}
+```
+
+</TabItem>
+<TabItem value="react">
+
+Declaration
+```html
+<rapid-radio-group>
+  <rapid-radio value="one">One</rapid-radio>
+  <rapid-radio value="two">Two</rapid-radio>
+  <rapid-radio value="three">Three</rapid-radio>
+</rapid-radio-group>
+```
+Usage
+```typescript
+export function MyComponent() {
+  const selectionChanged = (event) => {
+    console.log({ radioGroupValue })
```

Comment:
I think this can work like the angular example

`const target = event.target as HTMLInputElement; console.log(target.value);`